### PR TITLE
OBPIH-3276 Add header page for combined shipments

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy [hunk 1]
```diff
@@ -79,7 +79,11 @@ class StockMovementService {
             throw new ValidationException("Invalid stock movement", stockMovement.errors)
         }
 
-        return createRequisitionBasedStockMovement(stockMovement)
+        if(stockMovement.stockMovementType == StockMovementType.OUTBOUND) {
```
Comment (jmiranda) [inline grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy:82]: Will this change impact the existing workflows (order, inbound)?

**Review by jmiranda — COMMENTED** [review]
Looks good otherwise.

File: grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy (hunk 1 above)
Comment (DominikaHatala) [reply grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy:82]: It shouldn't be here. I have no idea why it's included, I was sure I've removed it but must have not added to commit. Will udpate soon.

Comment (DominikaHatala) [reply grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy]: @jmiranda I have updated PR.

File: src/js/components/stock-movement-wizard/StockMovementCombinedShipments.jsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+import SendMovementPage from './combined-shipments/SendMovementPage';
+import Wizard from '../wizard/Wizard';
+import apiClient from '../../utils/apiClient';
+import { showSpinner, hideSpinner, fetchTranslations, updateBreadcrumbs, fetchBreadcrumbsConfig } from '../../actions';
+import { translateWithDefaultMessage } from '../../utils/Translate';
+
+import './StockMovement.scss';
+
+/** Main combined shipments stock movement form's wizard component. */
+class StockMovements extends Component {
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/StockMovementCombinedShipments.jsx:17]: This should be named the same as file name to distinguish this from StockMovement.

**Review by awalkowiak — CHANGES_REQUESTED** [review]

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**